Dockerfile-bcoin

```diff
@@ -34,4 +34,7 @@ COPY --from=build /code/node_modules /code/node_modules
 COPY ./scripts/ /code/scripts
 
 ENTRYPOINT [ "node" ]
-CMD ["/code/scripts/bcoin-init.js"]
+
+# In order to have some predictability w/ docker instances
+# set the prefix via args to avoid any unexpected inconsistencies
+CMD ["/code/scripts/bcoin-init.js", "--prefix=/data"]
```

Comment:
Added based on conversations with @BluSyn. The volumes weren't persisting because of stray env I had in the regtest.bcoin.env file in `etc/` setting the prefix to `/code/.bcoin`. Having it in the image makes it a little more predictable but can still be overwritten via a custom `command` in docker-compose.